# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/browser/relations/browser.toml
# Commit: ae4f46dfec58
# Full Hash: ae4f46dfec5886a64797be7073f63bda296c3541
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2025-09-29 20:54 +0000
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Bug 1987945 - P1: Implement details relationship for CSS anchored local accessibles. r=Jamie,emilio
# ==============================================================================

diff -r 55449173b0e6 -r ae4f46dfec58 accessible/tests/browser/relations/browser.toml
--- a/accessible/tests/browser/relations/browser.toml	Mon Sep 29 20:39:23 2025 +0000
+++ b/accessible/tests/browser/relations/browser.toml	Mon Sep 29 20:54:29 2025 +0000
@@ -15,6 +15,8 @@
   "layout.css.anchor-positioning.enabled=true"
 ]
 
+["browser_anchor_positioning.js"]
+
 ["browser_popover_and_command.js"]
 
 ["browser_relations_general.js"]